docs/root/development/performance/cpu_battery_impact.rst

```diff
@@ -94,7 +94,7 @@ a process listening to traffic sent over this connection.
 
 Getting the build:
 
-1. Build the library using ``bazel build android_dist --config=android``
```

Comment:
Why don't we need this anymore? Still seems required?

Reply:
It's just for turning on logging, right?

Reply:
For now, yes, though that still seems like something we'd want by default

Reply:
@rebello95 what do we want by default? Logging?

Reply:
Yes, basically the configuration we get with `--config=android`

Reply:
Yeah we can add it back.

Reply:
added